### ISSUE TITLE
Implement monophyletic cluster growth checking

### ISSUE THREAD

**Issue by ConnorChato** [PoonLab/MountainPlot#6]
Currently, monophyletic clusters have growth defaulted to 0. This function (mono.pat.cluster) is masked from the user.

**Comment by ArtPoon** [comment]
@ConnorChato is this still an issue?

**Comment by ConnorChato** [comment]
Yes. Currently I'm not entirely sure we have a super great method of resolving growth for clusters that need to be monophyletic
